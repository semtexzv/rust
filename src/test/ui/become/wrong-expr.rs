#![feature(become_stmt)]

fn b() -> u32 {
    become b() + 1;
    //~^ ERROR become must always call a function
}

fn main() {
    println!("{}", b());
}
