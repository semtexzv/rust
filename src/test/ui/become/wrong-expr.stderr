error: become must always call a function
  --> $DIR/wrong-expr.rs:4:12
   |
LL |     become b() + 1;
   |            ^^^^^^^ this is not a function call

error: aborting due to previous error

